gslib/commands/rewrite.py

```diff
@@ -174,7 +174,8 @@
 
   -O            Rewrite objects with the bucket's default object ACL instead of
                 the existing object ACL. This is needed if you do not have
-                OWNER permission on the object.
+                OWNER permission on the object or if your bucket has uniform
+                bucket-level access (UBLA) enabled.
```

Comment:
In the case of UBLA, I'm assuming -O does something besides rewrite using the bucket's default object ACL, since my understanding is default object ACLs doin't exist in UBLA buckets? I'd recommend having two parts to this flag, something along the lines of:

"When a bucket has uniform bucket-level access (UBLA) enabled, the -O flag is required and [FILL IN THE BLANK REGARDING BEHAVIOR]. When a bucket has UBLA disabled, the -O flag rewrites objects with the bucket's default object ACL instead of the existing object ACL. This is needed if you do not have OWNER permission on the object."